.circleci/config.yml

```diff
@@ -78,6 +78,18 @@ jobs:
     steps:
       - checkout
       - run: make test-ci config=release default_pic=true
+  centos7-llvm-391-debug:
+    docker:
+      - image: ponylang/ponyc-ci:centos7-llvm-3.9.1
```

Comment:
It would be awesome if you would add a line with `user: pony` here, given the ci Dockerfile has this user as mentioned in the comment above. I tested these changes locally and they don't have any influence on the ci run.

I am sorry to bother you with this, but in case #2656 gets merged before this one doesn't need to be adapted again.

Reply:
Done.